pkg/api/controller.go

```diff
@@ -58,6 +58,13 @@ func NewController(config *config.Config) *Controller {
 	var controller Controller
 
 	logger := log.NewLogger(config.Log.Level, config.Log.Output)
+
+	if config.Cluster != nil {
+		// add global context of address:port for the instance's loggers only if
+		// scale-out cluster mode is enabled
+		host := fmt.Sprintf("%s:%s", config.HTTP.Address, config.HTTP.Port)
+		logger.Logger = logger.Logger.With().Str("host", host).Logger()
```

Comment:
Instead of "host", call this "clusterMember", so we know we are logging in context of a cluster.

Reply:
Addressed. I've renamed it to "clusterMember" and also added a new field for the member index for quicker lookups.